i18n/qgis_fr.ts

```diff
@@ -8164,7 +8165,7 @@ les paramètres de Processing.</translation>
     <message>
         <location filename="../python/plugins/processing/processing-i18n.cpp" line="107"/>
         <source>Computes emissivity from NDVI, generic method for sparse land. </source>
-        <translation>Calcule l&apos;emmissivité à partir des index NVDI, méthode générique pour les terrains pauvres.</translation>
+        <translation>Calcule l&apos;emissivité à partir des index NDVI, méthode générique pour les terrains pauvres. </translation>
```

Comment:
```suggestion
        <translation>Calcule l&apos;émissivité à partir des index NDVI, méthode générique pour les terrains pauvres. </translation>
```

Reply:
Fixed  (https://github.com/qgis/QGIS/pull/9145/files#diff-1488fb50ac041f325b8370f1687aa666R8168), thanks. Looks like github is not smart enough with such a big file to tag the suggestion as outdated.